src/main/common/filter.c

```diff
@@ -122,16 +128,25 @@ void biquadFilterInit(biquadFilter_t *filter, uint16_t filterFreq, uint32_t refr
     filter->a1 = a1 / a0;
     filter->a2 = a2 / a0;
 
-    // zero initial samples
-    filter->d1 = filter->d2 = 0;
+    filter->filterFreq = filterFreq;
+    filter->refreshRate = refreshRate;
+    filter->running = false;
 }
 
 // Computes a biquad_t filter on a sample
 float biquadFilterApply(biquadFilter_t *filter, float input)
 {
-    const float result = filter->b0 * input + filter->d1;
-    filter->d1 = filter->b1 * input - filter->a1 * result + filter->d2;
-    filter->d2 = filter->b2 * input - filter->a2 * result;
+    float result;
+    if (!filter->running) {
```

Comment:
Yes, servo stuff is executed in taskMainPidLoop and only if isServoOutputEnabled() returns true. 
Can I feel relatively free adding code to filterServos() in flight/servos.c?
Should I leave common/filter.c and .h unchanged at all cost (no additional higher level functions)?